src/plan_expand_hypertable.c

```diff
@@ -484,6 +572,16 @@ process_quals(Node *quals, CollectQualCtx *ctx, bool is_outer_join)
 			Expr *left = linitial(op->args);
 			Expr *right = lsecond(op->args);
 
+			/*
+			 * check for constraints with TIMESTAMPTZ OP INTERVAL calculations
+			 */
+			if ((IsA(left, Var) && IsA(right, OpExpr) &&
+				 castNode(OpExpr, right)->args->length == 2) ||
+				(IsA(left, OpExpr) && IsA(right, Var) && castNode(OpExpr, left)->args->length == 2))
+			{
+				qual = (Expr *) constify_timestamptz_op_interval(ctx->root, op);
+			}
+
```

Comment:
Nit: Split this into two separate if statements. There are two distinct cases where the constification should happen and IMHO it is good to make that clear, even if it happen to be identical operations for both cases.